### DISCUSSION TITLE
Modify right-click desktop menu?

### DISCUSSION THREAD

**Discussion by newbit1337** [lxqt/lxqt#2432] [Q&A] [answered]
Hi all, 
is there a way to add or remove entries from the menu that appears, when I right-click on the desktop?

I'm using Arch Linux + LXQt + Openbox.

Cheers and thanks for the great work!
newbit

**Comment by tsujan** [comment, marked as answer]
If by right click menu you mean Openbox's menu, it isn't related to LXQt. But if you mean pcmanfm-qt's Desktop, [custom actions](https://github.com/lxqt/pcmanfm-qt/wiki/custom_actions) can be added to it, like other folder menus.

**Comment by palonza** [comment]


**Reply by stefonarch** [reply]
This could well be an answer from ChatGpt: long, convincing  and lots of details but essentially all wrong.

**Reply by rajibando** [reply]
> @Stefonarch [This could well be an answer from ChatGpt: long, convincing and lots of details but essentially all wrong.](https://github.com/lxqt/lxqt/discussions/2432#discussioncomment-6015657)

Dear Mr. Stefonarch,
Your profile says that you are a maintainer. You say that the answer is all wrong.
Please apologise me for intruding on a post begun by someone else.
I use Debian and I am planning to install LXQt for my laptop.
I find that this question is one of the set of question that I am trying to find answer to, which was asked here:
[How can I add an arbitrary String with an associated Script/Command in the Right Click Drop Down Menu List in OpenBox WM? Also, autostart applications every session?](https://lists.debian.org/debian-user/2023/05/msg00909.html)
and in the OpenBox Mailing List. It appears that the OpenBox development and the Mailing List is as good as Dead.
The answer from tsujan, also a maintainer, is correct. lxqt can't solve OpenBox questions. But if it is for qt version of pcmanfm then the link posted appears correct, [as mentioned by tsujan:](https://github.com/lxqt/lxqt/discussions/2432#discussioncomment-6015371) "... But if you mean pcmanfm-qt's Desktop, [custom actions](https://github.com/lxqt/pcmanfm-qt/wiki/custom_actions) can be added to it, like other folder menus ...".
It would be very kind of you to order this post as well so that I could learn by example as to how to quote a poster by his username.
Best wishes

**Comment by stefonarch** [comment]
You have to distinguish right click menus in a pure openbox session from other right clicks on the desktop of LXQt or on files or folders.

Openbox has 2 right click menus: one on the desktop, "root menu" which can display applications to start and the right click on the title bar which provides window actions. Under LXQt with openbox the "root menu" isn't visible at all if you use the desktop module which normally is the case.

"Custom actions" are for the context menu of files/folders and items can be added to it as described. Those actions can be visible for some filetypes only (example image actions like resize for image files).

To quote just highlight the text and use the menu from the 3 dots "quote".

**Reply by rajibando** [reply]
Thank you, Mr. @stefonarch, for replying to my queries.

Ok, understood your explanation. But then why [the ChatGPT and "all wrong" comment? Couldn't understand the perspective.](https://github.com/lxqt/lxqt/discussions/2432#discussioncomment-6015657)

Regarding [@stefonarch: ...To quote just highlight the text and use the menu from the 3 dots "quote".](https://github.com/lxqt/lxqt/discussions/2432#discussioncomment-6057094) I meant the way I did just now. I didn't want to copy the full body. But thank you for indicating to the possible way.
